src/setuptools_pyproject_migration/__init__.py

```diff
@@ -201,6 +201,11 @@ def _generate(self) -> Pyproject:
         if classifiers:
             pyproject["project"]["classifiers"] = classifiers
 
+        description: str = dist.get_description()
+        # "UNKNOWN" is used by setuptools<62.2 when the description in setup.cfg is empty or absent
+        if description and description != "UNKNOWN":
```

Comment:
I'm beginning to wonder how many other places where they use this `"UNKNOWN"` sentinel value.  Looks like not just author information.

Reply:
Yeah apparently not. I wouldn't be surprised if it shows up in like half the fields... But I still think it's likely easier to just deal with them as we encounter them. Or at least I can't think of a cleaner way.